src/inject/dynamic-theme/adopted-style-manger.ts

```diff
@@ -0,0 +1,102 @@
+import {Theme} from '../../definitions';
+import {createStyleSheetModifier} from './stylesheet-modifier';
+import {forEach} from '../../utils/array';
+
+let adoptedStyleOverrides = new WeakMap<CSSStyleSheet, CSSStyleSheet>();
+let adoptedStyleOwners = [] as Array<ShadowRoot | Document>;
```

Comment:
This variable should become useless if you will collect style managers in `index.ts` file.